src/Abp/Domain/Repositories/AbpRepositoryBase.cs

```diff
@@ -51,7 +64,7 @@ public virtual List<TEntity> GetAllList()
             return GetAll().ToList();
         }
 
-        public virtual Task<List<TEntity>> GetAllListAsync()
+        public virtual Task<List<TEntity>> GetAllListAsync(CancellationToken cancellationToken = default)
```

Comment:
`AbpRespositoryBase` methods accept `cancellationToken` but do not forward or use it?

Reply:
Thanks for your input Aaron, I appreciate that. 
It doesn't need to forward `cancellationToken` since it is ORM indepented defined. However it is already overriden at;

- Abp.EntityFramework.Repositories.EfRepositoryBase
- Abp.EntityFrameworkCore.Repositories.EfCoreRepositoryBase
- Abp.NHibernate.Repositories.NhRepositoryBase

So the ORM can handle it.